Tests/CategoriesControllerTest.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using WebNetSample.Business.Abstract;
+using WebNetSample.Entity.Concrete;
+using WebNetSample.Tests.Configurations;
+using WebNetSample.WebNetMVC.Controllers;
+
+namespace WebNetSample.Tests;
+
+public class CategoriesControllerTest
+{
+    [Theory, AutoMoqData]
+    public void Index_Should_Return_As_Expected(List<Category> expected)
```

Comment:
Not really the same template I shared with you, underscore should be only between name of method and rest of description